packages/kit/src/nodash/internal.ts

```diff
@@ -38,6 +38,24 @@ export function lowerFirst(value?: string): Optional<string> {
   return value && value.charAt(0).toLowerCase() + value.slice(1);
 }
 
+/**
+ * Formats a camel case style `string` into a title case.
+ *
+ * @param text Text to transform.
+ */
+export function camelCaseToTitleCase(str: string): string;
+export function camelCaseToTitleCase(str?: string): Optional<string>;
```

Comment:
Why even allow str to be optional?

Reply:
Good I'll remove the option both as param and return values.